src/assignments/mars-rover.rst

```diff
@@ -0,0 +1,137 @@
+Assignment #3: Mars Rover
+=========================
+This task is going to put your unit tests, modules, and exceptions knowledge to use
+by writing tests and classes for the Mars rover named Curiosity.
+
+.. figure:: figures/curiosity-rover-selfie.jpg
+   :alt: Image of Curiosity rover taken by the rover on Mars.
+
+   Selfie of the Curiosity on Mars.
+
+
+Requirements
+------------
+
+#. Fork the `Mars rover starter repl.it <https://repl.it/@launchcode/mars-rover-starter>`_
+#. Write a unit test for each item in the Test List shown below
+
+   * Some tests have been created for you as examples
+
+#. Create or update these classes: `Rover`, `Message`, `Command`
+
+   * Details for each class are below
+
+#. Use modules
+
+   * Each class should be defined in it's own file and exported and imported using modules.
+
```

Comment:
The layout of this page is a bit much. Maybe instead of scrolling up and down all of the time, move the details below each step of the assignment?

Reply:
In my opinion that would still require scrolling. I really like the current layout, but can be talked out of it with more discussion.